### PR TITLE
Spelling src ci

### PR DESCRIPTION
**Pull request by jsoref**
The various src/* items seem slightly disparate, so I'm doing src/* individually.

split from https://github.com/rust-lang/rust/pull/110392

### CODE REVIEW

File: src/ci/docker/README.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ For targets: `armv7-unknown-linux-gnueabihf`
 (\*) These options have been selected to match the configuration of the arm
       toolchains shipped with Ubuntu 15.10
 (+) These options have been selected to match the gcc flags we use to compile C
-    libraries like jemalloc. See the mk/cfg/arm(v7)-uknown-linux-gnueabi{,hf}.mk
+    libraries like jemalloc. See the mk/cfg/arm(v7)-unknown-linux-gnueabi{,hf}.mk
```
Comment (jsoref) [inline src/ci/docker/README.md:214]: notable

File: src/ci/docker/host-x86_64/mingw-check/validate-toolstate.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# A quick smoke test to make sure publish_tooolstate.py works.
+# A quick smoke test to make sure publish_toolstate.py works.
```
Comment (jsoref) [inline src/ci/docker/host-x86_64/mingw-check/validate-toolstate.sh:2]: notable

**Comment by albertlarsan68** [comment]
LGTM
@bors rollup
r=me once squashed

**Comment by jsoref** [comment]
@albertlarsan68: are you waiting for something from me?

**Comment by albertlarsan68** [comment]
Thanks for the PR!
@bors r+ rollup

**Pull request merged**